specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyTrackedResources.json

```diff
@@ -379,6 +383,10 @@
       "enum": [
         "Microsoft.Management"
       ],
+      "x-ms-enum": {
+        "name": "ManagementGroupsNamespaceType",
```

Comment:
why did you end up making this (and a few of the other RP name occurrences) modelAsString: true but not in remediations or policyStates?  I thought the decision was to keep the "static" RP naming enums as modelAsString false because of some SDK breaking change concerns and only have the "default"/"latest" enums be modelAsString: true since those could technically evolve in the future

Reply:
Hey @pilor, I just updated the code to make all the namespace enums false. Please take a look again